UPDATING.md

```diff
@@ -66,6 +66,14 @@ https://developers.google.com/style/inclusive-documentation
 
 Now use NULL as default value for dag.description in dag table
 
+### Restrict editing DagRun State in the old UI (Flask-admin based UI)
+
+Before 1.10.11 it was possible to edit DagRun State in the `/admin/dagrun/` page
+ to any text.
+
+From Airflow 1.10.11, you would still be able to set from dropdown to a pre-defined option
+ but would not allow setting it to any text.
```

Comment:
I don't like the wording though, can you help me with that Ash?

I can include screenshots if that is more helpful

Reply:
```suggestion
Before 1.10.11 it was possible to edit DagRun State in the `/admin/dagrun/` page
 to any text.

In Airflow 1.10.11+, the user can only choose the states from the list.
```